docs/manual/guides/formular-speichern.de.md

```diff
@@ -0,0 +1,550 @@
+
+{{% notice info %}}
+Zur Auswahl der Zieltabelle stehen alle vorhandenen Datenbanktabellen deiner Contao Installation zur Verfügung. 
+Du könntest dir auch eine eigene Datenbanktabelle erstellen. Der entsprechende Tabellenname muss dann mit dem Präfix »tl_«
+beginnen. Grundsätzlich müssen die Feldnamen deines Formulars mit den entsprechenden Feldnamen der Datenbanktabelle 
+übereinstimmen.
+{{% /notice %}}
+
+
+### Der Hook prepareFormData
+
+Die Datenbanktabelle ``tl_calendar_events`` beinhaltet zahlreiche Felder. Nicht alle sind für unser Beispiel notwendig.
+Wir benötigen zumindest Einträge für die Felder »pid«, »title«, »alias«, »author«, »startDate«, »startTime«, »endDate«, 
+»endTime« und »published«, damit wir im Contao Backend eine gültige Darstellung unserer Reservierungen erhalten. Optional 
+wären Einträge für »location«, »description« oder »teaser«. 
+
+Unser Formular bedient zur Zeit zumindest die Felder »title«, »startDate« und optionales »endDate«. Die weiteren Felder
+könnte man über versteckte Formularfelder hinzufügen. Diese Vorgehensweise ermöglicht jedoch die Manipulation der zu 
+übermittelten Formulardaten. Aus diesem Grund verwenden wir den Hook 
+»[prepareFormData](https://docs.contao.org/dev/reference/hooks/prepareFormData/)«.
+
+
+Dieser Hook greift nach der Formular Übermittlung aber vor der eigentlichen Datenspeicherung. Du kannst darüber 
+die übermittelten Daten bearbeiten und neue Informationen hinzufügen. Erstelle dir in deiner Contao Installation 
+die Verzeichnisse ``src\EventListener\`` und darin die Datei ``PrepareFormDataListener.php`` mit folgendem Inhalt:
+
+
+```php
+// src/EventListener/PrepareFormDataListener.php
+
+namespace App\EventListener;
+
+use Contao\CoreBundle\ServiceAnnotation\Hook;
+use Contao\Form;
+use Contao\CoreBundle\Slug\Slug;
+
+/**
+ * @Hook("prepareFormData")
+ */
+class PrepareFormDataListener 
+{
+  private $slug;
+
+  public function __construct(Slug $slug)
+  {
+      $this->slug = $slug;
+  }  
+
+  public function __invoke(array &$submittedData, array $labels, array $fields, Form $form): void
+  {
+    // set id of form, eventarchiv and author
+    $idForm         = 3;
+    $idEventArchiv  = 5;
+    $idAuthor       = 1;
+    
+    $ticketNr = strtotime("now");
+    
+    // form restriction 
+    if ($form->id == strval($idForm)) {
+
+      // mandatory fields
+      $submittedData['pid']       = $idEventArchiv;
+      $submittedData['author']    = $idAuthor;
+      $submittedData['published'] = 1;
+      
+      $tmpTitle = $submittedData['title'] . " Ticket: " . $ticketNr;
+      $submittedData['title'] = $tmpTitle;
+      $submittedData['alias'] = $this->slug->generate(
+        $tmpTitle, ['validChars' => 'a-z0-9', 'locale' => 'de', 'delimiter' => '-']
+      );      
```

Comment:
This will cause an error, if two users submit the form in the same second without the duplicate check callback. Since you want "ticket numbers" a better solution would be to create a table lock and then create a ticket number based on the previous ticket number.